tests/spec/fs.link.spec.js

```diff
@@ -110,4 +109,18 @@ describe('fs.link', function() {
       });
     });
   });
+
+});
+
+describe('fs.promises.link', function() {
+
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should return a promise', function() {
+    var fs = util.fs().promises;
```

Comment:
Maybe consider changing the `fs` variable to `fsPromise` to differentiate it from the non-promise FileSystem?

Reply:
Thanks for the feedback! I agreed with it and decided to change the variable's name to make it more accurate.